en/docs/ai-workspace/1.0.0/llm-proxies/configure-proxy.md

```
@@ -0,0 +1,153 @@
+
+## Get started
+
+Once the proxy is deployed, the proxy details page shows the **Get Started** panel on the right.
+
+### Invoke URL
+
+Select a gateway from the **Gateways** dropdown to see the base URL for calling this proxy through that gateway.
+
+The URL follows the format:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)